README.md

```diff
@@ -1,8 +1,12 @@
-# Archive of Meeting Notes from Hackmd.io
+# Meeting Notes
 
-Active notes are captured at [Notary Project Meeting Notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view)
+This repository is in active maintenance and contains the archive of meeting notes for Notary Project [community meetings](https://notaryproject.dev/community/#community-meetings).
```

Comment:
nit:
...for _the_ Notary Project

Reply:
Thanks @SteveLasker , I will create a new issue/pr to fix it, see https://github.com/notaryproject/meeting-notes/issues/19